pinot-tools/src/test/resources/memory_estimation/data-characteristics.json

```diff
@@ -0,0 +1,37 @@
+{
+  "numberOfRows": 10000,
+  "columnCharacteristics": [
```

Comment:
This is not same as what is taken by RecommenderDriver right? I don't think it takes columnCharacteristics.  See the file InputManager and SchemaWithMetadata.

Reply:
It is fine to extend that format but hereI feel we are looking at a different format

Reply:
https://github.com/apache/incubator-pinot/blob/master/pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/InputManager.java

https://github.com/apache/incubator-pinot/blob/master/pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/metadata/SchemaWithMetaData.java